src/net_processing.cpp

```diff
@@ -345,10 +353,7 @@ class PeerManagerImpl final : public PeerManager
      * their own locks.
      */
     std::map<NodeId, PeerRef> m_peer_map GUARDED_BY(m_peer_mutex);
-};
-} // namespace
 
-namespace {
     /** Number of nodes with fSyncStarted. */
     int nSyncStarted GUARDED_BY(cs_main) = 0;
```

Comment:
You're renaming the other data members as you move them into `PeerManagerImpl`. Perhaps rename this to `m_sync_started`?

Reply:
I'm only renaming the ones that are currently marked `g_` since that would be wrong once they're no longer global. I'm avoiding renaming the others because that's unnecessary churn.

I think all of them are `GUARDED_BY(cs_main)` so renaming them at the same time as switching them to a new mutex would make sense -- that's the point at which you need to be reviewing every place the variable is touched to ensure the locking makes sense, so having every place the variable is touched show up in the diff because it's getting renamed would make sense.

Reply:
ok!